modules/post/windows/gather/credentials/minio_client.rb

```diff
@@ -0,0 +1,117 @@
+##
+# This module requires Metasploit: https://metasploit.com/download
+# Current source: https://github.com/rapid7/metasploit-framework
+#
+# @blurbdust based this code off of https://github.com/rapid7/metasploit-framework/blob/master/modules/post/windows/gather/credentials/gpp.rb
+# and https://github.com/rapid7/metasploit-framework/blob/master/modules/post/windows/gather/enum_ms_product_keys.rb
+##
+
+class MetasploitModule < Msf::Post
+  include Msf::Post::Windows::Registry
+  include Msf::Post::Windows::UserProfiles
+
+  def initialize(info = {})
+    super(
+      update_info(
+        info,
+        'Name' => 'Windows Gather MinIO Client Key',
+        'Description' => %q{
+          This is a module that searches for MinIO Client credentials on a windows remote host.
+        },
+        'License' => MSF_LICENSE,
+        'References' => [
+          [ 'URL', 'https://blog.kali-team.cn/Metasploit-MinIO-Client-7d940c60ae8545aeaa29c96536dda855' ]
+        ],
+        'Author' => ['Kali-Team <kali-team[at]qq.com>'],
+        'Platform' => [ 'win' ],
+        'SessionTypes' => [ 'meterpreter' ],
```

Comment:
Is there any reason this module would not also work on shell and PowerShell sessions? I think `grab_user_profiles` should work on these sessions.

```suggestion
        'SessionTypes' => %w[meterpreter powershell shell],
```